### PR TITLE
fix RD-3139

### PR DESCRIPTION
**Pull request by daniellin215**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (grekinsky) [inline package.json]: Why create package.json for this?

Comment (daniellin215) [reply package.json]: Edgar helped make this so I can just run 'npm run devSetup' and it'll link all the repos, it was in there the last version I accidentally removed it.

Comment (grekinsky) [reply package.json]: lol, ok yeah that really comes handy, is this documented in the readme?

Comment (daniellin215) [reply package.json]: yup it is

File: tpp/src/main/Representable.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // @flow
```
Comment (grekinsky) [inline tpp/src/main/Representable.js]: You should bump tpp version afaik

Comment (daniellin215) [reply tpp/src/main/Representable.js]: oh ya, the version was bumped again. I will bump it to 1.0.12

**Pull request merged**